adding initial fastlane structure

IzzySoft opened the pull request:
This will be helpful…

* if you wish to bring your app to [F-Droid](https://f-droid.org/)
* for deploying your app to Google Play (this might need the Fastlane binaries)

Further, you might wish to move your screenshots from `assets/` to `fastlane/metadata/android/en-US/images/phoneScreenshots/` so they get picked up as well. I didn't do this with my PR here as I cannot tell what else (except for the ReadMe, which I could have easily adjusted) might depend on the original location :wink:

itsarjunsinh commented:
F-Droid availabilty has been part of my to-do list from the start. Since I don't have time at the moment to add Elder Launcher in the main F-droid repo, I'm keen on adding it through your third-party repo. Thanks for this PR.

After merging this, and once I move the screenshots is there anything else you require for the app to show up in your app repo?

IzzySoft commented:
Well, it already [is in my repo](https://apt.izzysoft.de/fdroid/index/apk/xyz.arjunsinh.elderlauncher) – so if you want to [place a badge](https://gitlab.com/IzzyOnDroid/repo/#if-my-app-is-available-here-do-you-have-a-badge-i-can-use-to-link-to-it) next to the one of Google Play, be welcome :smile:

Actually, this PR contains the Fastlane data I've currently setup locally for that. Just thought it a good idea to rather have it "upstream" in the app's repo, so you can adjust it easily when needed :wink: Just let me know once you've moved the screenshots into the Fastlane structure, so I adjust the config here to check them as well with each new version of the app released.

itsarjunsinh commented:
I've moved the screenshots. Thanks again for the PR.

IzzySoft commented:
Thanks for merging! But the screenshots need to be moved one level deeper, inside `images/phoneScreenshots` – see [my cheat-sheet](https://gitlab.com/snippets/1895688) for a quick reference. Your green feature graphic (I see one [here](https://www.appbrain.com/app/xyz.arjunsinh.elderlauncher)) you could place as `images/featureGraphic.png` if you wish.

itsarjunsinh commented:
Done. The cheat-sheet was helpful. I was actually trying to search for something like it before I made the previous commit.

IzzySoft commented:
Apologies, my bad – I should have mentioned it together with the screenshots before. I usually do, so it must have slipped through – I was working on 5 other such PRs and must have lost track… Thanks for your patience – and for adding the badge :smiley:

Wow, nice featureGraphic you got there – I replaced the old one right on the spot! Also updated the config, so from now on changes you make to the existing files will be picked automatically whenever there's a new release. Should you decide to add other types (like, per-release changelogs), please let me know so I enable them as well.

Now: all the best – and thanks again!
